core/src/Kokkos_Macros.hpp

```diff
@@ -153,6 +153,13 @@
   __clang_major__ * 100 + __clang_minor__ * 10 + __clang_patchlevel__
 #endif
 
+// Macro for the Cray compiler based on clang.
+// Different from classical Cray wihch is defined above.
```

Comment:
```suggestion
// Different from classical Cray which is defined above.
```
Is it a problem that the compiler now identifies as both `CLANG` and `CRAYCLANG`?